### ISSUE TITLE
[Frontend] Update Primary Nav to Use Nav Component

### ISSUE THREAD

**Issue by nathanielarking** [VerdanTech/Vdt-Webapp#59]
The PrimaryNav currently uses a tooltip to get hoverable submenus. This is not keyboard accessible however. Upon release of bits for svelte 5, the new nav component should be used.
